tools/CouchDB_MRI_Importer.php

```diff
@@ -102,7 +102,7 @@ function _generateCandidatesQuery($ScanTypes)
             LEFT JOIN feedback_mri_comments fmric
             ON (fmric.CommentTypeID=7 AND fmric.SessionID=s.ID)
             WHERE c.Entity_type != 'Scanner' AND c.PSCID NOT LIKE '%9999'
-                  AND c.Active='Y' AND s.Active='Y' AND c.CenterID <> 1";
+                  AND c.Active='Y' AND s.Active='Y' AND c.CenterID <> 1 ORDER BY s.ID";
```

Comment:
I don't think this is the right place to add an `ORDER BY` to fix the problem of document's fields being out of order. It's the query in getScanTypes() that is being inconsistent (as far as I can tell) which is causing it to generate different documents for the same data depending on MySQL's mood. The order in which sessions are processed is irrelevant.